<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>resultwindow</class>
 <widget class="QDialog" name="resultwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-radius: 10px;
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>520</width>
     <height>124</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0px;
background-color: rgb(234, 216, 24);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>101</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3_sw">
     <item>
      <widget class="QLabel" name="speed_typing_label_sw">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>123</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ЗН./ХВ</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>101</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2_sw">
     <item>
      <widget class="QLabel" name="counter_Label">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="text_under_counter">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>ПОМИЛОК</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>

  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>521</width>
     <height>77</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_sw">
    <item>
     <widget class="QLabel" name="main_label">
      <property name="font">
       <font>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(41, 69, 85);</string>
      </property>
      <property name="text">
       <string>Вправу завершено!</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="instruct_label">
      <property name="styleSheet">
       <string notr="true">color: rgb(148, 162, 170);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>521</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="againButton">
      <property name="minimumSize">
       <size>
        <width>97</width>
        <height>44</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>97</width>
        <height>44</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Verdana</family>
        <pointsize>11</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(255, 222, 14);
border: 3px solid rgb(170, 255, 255);
color: rgb(85, 85, 255);
border-radius: 5px;
}
QPushButton:hover{
border: 3px solid rgb(168, 168, 168);
}</string>
      </property>
      <property name="text">
       <string>ЩЕ РАЗ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextLevelButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>78</width>
        <height>44</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>78</width>
        <height>44</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
border: 3px solid  rgb(163, 245, 245);
border-radius: 5px;
color: rgb(85, 170, 255);
background-color: rgb(255, 222, 14);
}
QPushButton:hover{
border: 3px solid rgb(168, 168, 168);
}</string>
      </property>
      <property name="text">
       <string>ДАЛІ</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
